c.pod_design.md

```diff
@@ -215,13 +215,7 @@ kubernetes.io > Documentation > Concepts > Workloads > Controllers > [Deployment
 <p>
 
 ```bash
-kubectl create deployment nginx  --image=nginx:1.7.8  --dry-run=client -o yaml > deploy.yaml
-vi deploy.yaml
-# change the replicas field from 1 to 2
-# add this section to the container spec and save the deploy.yaml file
-# ports:
-#   - containerPort: 80
-kubectl apply -f deploy.yaml
+k run nginx --image=nginx:1.7.8 --replicas=2 --port=80
```

Comment:
this gives a warning: `Flag --replicas has been deprecated, has no effect and will be removed in the future.`